doc/devel/development_setup.rst

```diff
@@ -60,6 +60,15 @@ true for ``*.py`` files.  If you change the C-extension source (which might
 also happen if you change branches) you will have to re-run
 ``python -m pip install -ve .``
 
+Installing pre-commit hooks
+===========================
+You can optionally install `pre-commit <https://pre-commit.com/>`_ hooks.
+These will automatically check flake8 and other style issues when you run
+``git commit``. To install the hooks ::
+
+    pip install pre-commit
+    pre-commit install
+
```

Comment:
Can you add a sentence about where the config file is so devs can modify their local version?

Reply:
I added something, let me know if that's what you were looking for